.github/mergify.yml

```diff
@@ -3,8 +3,9 @@ pull_request_rules:
     conditions:
       - author~=^dependabot\[bot\]$
       - check-success~=lint
+      - check-success~=build
       - check-success~=test
-      - dependabot-update-type = version-update:semver-minor
+      - -dependabot-update-type = version-update:semver-major
```

Comment:
wait am a bit confused, should this not be minor then?

Reply:
<img width="830" alt="Screenshot 2024-04-09 at 1 35 38 PM" src="https://github.com/isomerpages/isomercms-frontend/assets/44049504/4d1318e1-d9dd-4ffe-b1e9-bf62b155b937">

didn't duplicate the comment here but this negates the condition so this is essentially everything but major (minor + patches)